src/pybind/mgr/dashboard/frontend/e2e/pools/pools.po.ts

```diff
@@ -61,13 +77,17 @@ export class PoolPageHelper extends PageHelper {
             return $('li.delete a') // click on "delete" menu item
               .click()
               .then(() => {
-                const getConfirmationCheckbox = () => $('#confirmation');
                 browser
-                  .wait(() => EC.visibilityOf(getConfirmationCheckbox()), Helper.TIMEOUT)
+                  .wait(Helper.EC.visibilityOf($('.custom-control-label')), Helper.TIMEOUT)
                   .then(() => {
-                    this.moveClick(getConfirmationCheckbox()).then(() => {
-                      return element(by.cssContainingText('button', 'Delete Pool')).click(); // Click Delete item
-                    });
+                    const getConfirmationCheckbox = () => $('.custom-control-label');
+                    browser
+                      .wait(() => EC.visibilityOf(getConfirmationCheckbox()), Helper.TIMEOUT)
```

Comment:
To be consistent with all the others.
```suggestion
                      .wait(Helper.EC.visibilityOf(getConfirmationCheckbox()), Helper.TIMEOUT)
```